Sources/PostgresNIO/Connection/PostgresConnection+Configuration.swift

```diff
@@ -193,8 +193,8 @@ extension PostgresConnection {
         ///   - channel: The `NIOCore/Channel` to use. The channel must already be active and connected to an
         ///     endpoint (i.e. `NIOCore/Channel/isActive` must be `true`).
         ///   - tls: The TLS mode to use. Defaults to ``TLS-swift.struct/disable``.
-        public init(establishedChannel channel: Channel, username: String, password: String?, database: String?) {
-            self.init(endpointInfo: .configureChannel(channel), tls: .disable, username: username, password: password, database: database)
+        public init(establishedChannel channel: Channel, tls: PostgresConnection.Configuration.TLS = .disable, username: String, password: String?, database: String?) {
```

Comment:
This is a breaking API change. Would you mind adding an additional initializer? This initializer should then call the new one that you just created.

Do you feel comfortable to add a unit test using a `NIOAsyncTestingChannel`? An example can be found here: 
https://github.com/vapor/postgres-nio/blob/f2a6394a2e7157d547727b975fc0328b92f89fb1/Tests/PostgresNIOTests/New/PostgresConnectionTests.swift#L41C1-L82C1

Reply:
I probably won't have time to do the test right now, as I need to fix the wrong channel handler order first for TLS 🤭

Reply:
> I need to fix the wrong channel handler order first for TLS

Not sure I follow here.

Reply:
See #527